### PR TITLE
added some useful links and few questions

### PR DESCRIPTION
**Pull request by benjithorpe**

### CODE REVIEW

File: useful-links/useful-links.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="useful-links.css" />
     <title>Useful Links</title>
   </head>
+
```
Comment (DavidMatthewFraser) [inline useful-links/useful-links.html]: we do not use white space between lines of code

File: useful-links/useful-links.html [hunk 2]
```diff
@@ -104,18 +106,18 @@ <h3>Podcasts</h3>
...
+          <li><a href="https://www.topcoder.com/" target="_blank">TopCoder</a> solve algorithms, competitive programming</li> <!-- Share your information here -->
           <li>.</li> <!-- Share your information here -->
         </ul>
       </div>
       <div class="list">
         <h3>Mobile apps</h3>
         <ul>
           <li><a href="https://www.programming-hero.com/" target="_blank">Programming Hero</a> Beginner friendly</li>
-          <li><a href="https://play.google.com/store/apps/details?id=com.sololearn" target="_blank">sololearn</a> - code playground, learning, coding challenges</li>
+          <li><a href="https://play.google.com/store/apps/details?id=com.sololearn" target="_blank">SoloLearn</a> - code playground, learning, coding challenges</li>
```
Comment (DavidMatthewFraser) [inline useful-links/useful-links.html]: good catch

File: quiz/quiz-questions.js [hunk 3]
```diff
@@ -67,19 +67,29 @@ const htmlQuestions = [
...
   },
   {
     question: "In which element would I usually find <footer>?",
     answers: ["<header>", "<body>", "<head>", "<nav>"],
-    correctAnswer: "<body>"
-  }
+    correctAnswer: "<body>",
+  },
+  {
+    question: "Which of the following can never contain an attribute?",
```
Comment (DavidMatthewFraser) [inline quiz/quiz-questions.js:84]: cool question 👍

**Comment by DavidMatthewFraser** [comment]
Thanks for working on the quiz + useful-links!

File: useful-links/useful-links.html (hunk 1 above)
Comment (benjithorpe) [reply useful-links/useful-links.html]: Ohh, my bad, I've use to seeing white spaces :)

**Pull request merged**